it/uk/gov/hmrc/incorporatedentityidentificationfrontend/controllers/CaptureCompanyNumberControllerISpec.scala

```diff
@@ -39,13 +40,7 @@ class CaptureCompanyNumberControllerISpec extends ComponentSpecHelper
       await(insertJourneyConfig(
         journeyId = testJourneyId,
         authInternalId = testInternalId,
-        continueUrl = testContinueUrl,
-        optServiceName = None,
-        deskProServiceId = testDeskProServiceId,
-        signOutUrl = testSignOutUrl,
-        businessEntity = LimitedCompany,
-        businessVerificationCheck = true
-      ))
+        journeyConfig = testLimitedCompanyJourneyConfig))
```

Comment:
I'm glad you've made this change, because I have too. I was assuming you hadn't and I was going to have an interesting merge to content with.